CHANGELOG.rst

```diff
@@ -1,6 +1,60 @@
 
+0.6.0 (2018-09-10)
+------------------
+
+MLflow 0.6.0 introduces two major features:
+
+- A Java client API, published to Maven
+- Inherent support for saving and serving SparkML models as MLeap for low-latency serving
+
+In addition to these features, there are a host of improvements and bugfixes to the REST API, Python API, tracking UI, and documentation.
+
+Breaking changes:
+
+We fixed a few inconsistencies in the the ``mlflow.tracking.MlflowService`` introduced in 0.5.0:
+
+- ``MlflowService.list_runs`` was changed to ``MlflowService.list_run_infos`` to reflect the information actually returned by the call. It now returns a ``RunInfo`` instead of a ``Run`` (#334, @aarondav)
+- ``MlflowService.log_artifact`` and ``MlflowService.log_artifacts`` now take a ``run_id`` instead of ``artifact_uri``. This now matches ``list_artifacts`` and ``download_artifacts``  (#444, @aarondav)
+
+Features:
+
+- Java client API added with support for the MLflow Tracking API (analogous to ``mlflow.tracking``), allowing users to create and manage experiments, runs, and artifacts. The release includes a `usage example <https://github.com/mlflow/mlflow/blob/master/mlflow/java/client/src/main/java/org/mlflow/tracking/samples/QuickStartDriver.java>`_ and `Javadocs <https://mlflow.org/docs/latest/java_api/index.html>`_. The client is published to Maven under `mlflow:mlflow <https://mvnrepository.com/artifact/mlflow/mlflow>`_ (#380, #394, #398, #409, #410, #430, #452, @aarondav)
+- SparkML models are now saved in MLeap format (https://github.com/combust/mleap), when applicable. Model serving platforms can choose to serve using this format instead to dramatically decrease prediction latency. SageMaker now does this by default (#324, #327, #331, #395, #428, #435, #438, @dbczumar)
+- [API] Experiments can now be deleted and restored via REST API, Python Tracking API, and MLflow CLI (#340, #344, #367, @mparkhe)
+- [API] Tags can now be set via a SetTag API, and they have been moved to ``RunData`` from ``RunInfo`` (#342, @aarondav)
+- [API] Added ``list_artifacts`` and ``download_artifacts`` to ``MlflowService`` to interact with a run's artifactory (#350, @andrewmchen)
+- [API] Added ``get_experiment_by_name`` to Python Tracking API, and equivalent to Java API (#373, @vfdev-5)
+- [API/Python] Version is now exposed via ``mlflow.__version__``.
+- [API/CLI] Added ``mlflow artifacts`` CLI to list, download, and upload to run artifact repositories (#391, @aarondav)
+- [UI] Added icons to source names in MLflow Experiments UI (#381, @andrewmchen)
+- [UI] Added support to view ``.log`` and ``.tsv`` files from MLflow artifacts UI (#393, @Shenggan; #433, @whiletruelearn)
+- [UI] Run names can now be edited from within the MLflow UI (#382, @smurching)
+- [Serving] Added ``-host`` option to ``mlflow serve`` to allow listening on non-local addressess (#401, @hamroune)
+- [Serving/SageMaker] Elastic Container Registry boto client can now take an AWS region (#366, @dbczumar)
```

Comment:
I'm not sure we expose a boto client; I think this should say our API and CLI can now take an AWS region